client/assets/stylesheets/shared/_typography.scss

```diff
@@ -7,11 +8,7 @@ $code: Monaco, Consolas, 'Andale Mono', 'DejaVu Sans Mono', $monospace;
 $serif-fallback: Georgia, 'Times New Roman', Times, serif;
 $serif: 'Noto Serif', $serif-fallback;
 
-$sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell',
-	'Helvetica Neue', sans-serif;
-$sans-rtl: Tahoma, $sans;
```

Comment:
Extracting `sans` and leaving the rest of these general font declarations seems strange. Do you think `$monospace`, `$code`, `$serif`, and `$serif-fallback` should move as well?

Reply:
I'm keen to move them when `@automattic/components` needs them. I'd prefer to keep Calypso specific code out of `@automattic/components` in order to keep what's exposed by the package minimal, leaving the code free to change and iterate as necessary in Calypso.

Reply:
I also like the idea of keeping `$monospace` etc. where is until it's actually shared by something. It could be that those fonts are never get used by a shared component, and that's fine.